reactor-core/src/main/java/reactor/core/scheduler/Schedulers.java

```diff
@@ -215,9 +217,21 @@ public static Scheduler fromExecutorService(ExecutorService executorService, Str
 	 * between callers. They can however be all {@link #shutdownNow() shut down} together, or replaced by a
 	 * {@link #setFactory(Factory) change in Factory}.
 	 *
+	 * <p>
+	 * Note: Starting from 3.6.0 within Java 21 runtime with
+	 * {@link #DEFAULT_BOUNDED_ELASTIC_ON_VIRTUAL_THREADS} set to {@code true} the
+	 * returned scheduler will run on {@link VirtualThread}. Consider increasing
+	 * {@link #DEFAULT_BOUNDED_ELASTIC_SIZE} to run more concurrent
+	 * {@link VirtualThread} underneath.
+	 *
+	 *
 	 * @return the common <em>boundedElastic</em> instance, a {@link Scheduler} that dynamically creates workers with
 	 * an upper bound to the number of backing threads and after that on the number of enqueued tasks, that reuses
-	 * threads and evict idle ones
+	 * threads and evict idle ones. If runs on Java 21 with
```

Comment:
The `@return` section needn't be so detailed. It can just mention that a shared global boundedElastic instance is created and returned instead of repeating notes from the above.